### DISCUSSION TITLE
Y-axens intervaller og max

### DISCUSSION THREAD

**Discussion by petermandnielsen** [rndfm/elspotpris#110] [Ideas]
Vile det ikke være en god idé, hvis der var mulighed for selv at kunne definere et fast max. på y-aksen, fx prisen på elselskabets nuværende fastprisaftale?

**Comment by rndfm** [comment]
En anden fordel med en fast max på y-aksen vil være at det vil blive nemmere at se generel høj eller lave priser.
Lige nu ser 48timer med 2kr ens ud med 48timer med 8kr.

Hvorfor ville man låse aksen til en fastprisaftale?

**Reply by petermandnielsen** [reply]
Det forhold, du beskriver med generel høj eller lav pris, var faktisk også anledningen til mit indlæg. 
At kunne markere/definere prisen på en fast aftale er blot for hurtigt at kunne se, om det var en god beslutning af vælge varibel pris eller om man skal ærgre sig resten af dagen ;o)

**Reply by rndfm** [reply]
Okay. Måske man skulle lave en mulighed for at lave en horisontal markering på grafen. Her kunne man så indsætte sin fastpris?

**Reply by petermandnielsen** [reply]
Enig, men visuelt er det nok vigtigere at ændre y-aksen som du selv foreslår.

**Reply by rndfm** [reply]
Jeg har tumlet lidt med om man skulle lave minimum og maksimum ud fra noget kombineret.

F.eks. Hvis prisen på 48 timer svinger fra 6-8 kr. Så ser 6 kroner rigtigt billigt ud i øjeblikket. Især fordi 6kr søjlen bliver skriggrøn.
Så måske minimumsfarven kun skulle være skriggrøn hvis den er under minimum som beregnes ala (6 + 2) / 2 = 4 kr.
Altså så man vægter minimumsfarven mellem 6kr og en fast lav værdi (2kr)

Det samme med maksimum. Hvis 48timer spænd er 1,5 - 2 kr så ser 2 kr virkeligt dyrt ud da den bliver BLODRØD. Måske den også skulle farves ud fra samme vægtning. Altså f.eks. at max = (2 + 8) / 2 = 5kr. 

Måske man kunne lave vægtningen så aktuel pris og statiske min max bliver vægtet forskelligt.

**Reply by rndfm** [reply]
Jeg har lavet lidt ændringer så nu regnes max på grafen ud fra:
(max pris på 48timer * 0,8) + (8 * 0,2)

Altså en 80/20 vægtning mellem max prisen i perioden og en fast sats på 8kr.

Minimum på grafen bliver regnet på sammen måde(næsten):
(min pris på 48 timer * 0,8) + (1,5 * 0,2)

Både farverne og min/max på grafen justeres herefter.

Lad mig høre hvad du synes :)

**Reply by petermandnielsen** [reply]
Det fungerer fint. Farverne er nu mere retvisende. Thank's
